<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calcul Divers . . .</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="inverse">
      <attribute name="title">
       <string>Inverse</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>19</y>
         <width>661</width>
         <height>391</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="affichageInverse">
            <property name="text">
             <string>  Saisir ici le nombre à inverser : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="EditNombre"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="affichageResultInverse">
            <property name="text">
             <string>L'inverse de ce nombre est : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LabInverse">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="quotrest">
      <attribute name="title">
       <string>Quot Rest</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>20</y>
         <width>501</width>
         <height>381</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Saisir ici un numérateur :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="EditNum"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Saisir ici le numérateur : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="EditDen"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Quotient :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LabQuotient">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Reste :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LabReste">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="butValid">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>180</y>
         <width>91</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="minmax">
      <attribute name="title">
       <string>Min Max</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>29</x>
         <y>20</y>
         <width>631</width>
         <height>321</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Entrez quatres nombres, le programme vous retournera le maximum et le minium :</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLineEdit" name="Editn1"/>
          </item>
          <item>
           <widget class="QLineEdit" name="Editn2"/>
          </item>
          <item>
           <widget class="QLineEdit" name="Editn3"/>
          </item>
          <item>
           <widget class="QLineEdit" name="Editn4"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Minimum :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LabMini">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Maximum :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LabMaxi">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>360</y>
         <width>561</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
